lib/data/context.js

```diff
@@ -0,0 +1,50 @@
+const isGenerator = require('../utils').isGenerator;
+const DataTable = require('./table');
+
+module.exports = function (context) {
+
+  context.Data = function (dataTable) {
+    let data = detectDataType(dataTable);
+
+    return {
+      Scenario: function (title, opts, fn) {
+        if (typeof opts === 'function' && !fn) {
+          fn = opts;
+          opts = {};
+        }
+        data.forEach((dataRow) => {
+          opts.data = dataRow;
+          let test = context.Scenario(`${title} | ${dataRow}`, opts, fn);
+          test.inject.current = dataRow;
+        });
+      }
+    };
+  };
+
+  context.xData = function (dataTable) {
```

Comment:
Looks like it's important to have ability to launch only test that contains `Data`